script.js

```diff
@@ -1,19 +1,19 @@
-const display = document.querySelector("#display-data-done-main");
+const display = document.querySelector(".display-data-done-main");
 
 /* FETCH JSON DATA */
 
 
 const getData = () =>{
-  fetch('./user.json')
+  return fetch('./user.json')
```

Comment:
`fetch` is an asynchronous function. It doesn't return immediately because it makes a network call. `fetch` returns a promise. Have you worked with those before? We can return the promise out of the `getData` function